scripts/test_twitter_ingestion.py

```
@@ -0,0 +1,157 @@
+        response = requests.post(INGESTION_ENDPOINT, json=payload, timeout=30)
+        
+        if response.status_code == 200:
+            result = response.json()
+            if result.get("success", False):
+                mutations_generated = result.get("mutations_generated", 0)
+                mutations_executed = result.get("mutations_executed", 0)
+                
+                print(f"✅ {data_type}: {mutations_generated} mutations generated, {mutations_executed} executed")
+                
+                if mutations_generated > 0:
+                    print(f"   🎉 SUCCESS: Mutations are being generated!")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)